File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-jamba-development/JMBA_Olo_Android_SDK/app/src/main/java/com/wearehathway/apps/olo/Models/OloOrderStatus.java
package com.wearehathway.apps.olo.Models;

import java.util.ArrayList;

/**
 * Created by Nauman Afzaal on 28/04/15.
 */
public class OloOrderStatus
{
    private String id;
    private String timeplaced;
    private String vendorname;
    private String status;
    private float total;
    private float subtotal;
    private float salestax;
    private String orderref;
    private String readytime;
    private String vendorextref;
    private String deliverymode;
    private ArrayList<OloOrdersSatusProduct> products;
    private boolean iseditable;

    public String getId()
    {
        return id;
    }

    public String getTimeplaced()
    {
        return timeplaced;
    }

    public String getVendorname()
    {
        return vendorname;
    }

    public String getStatus()
    {
        return status;
    }

    public float getTotal()
    {
        return total;
    }

    public float getSubtotal()
    {
        return subtotal;
    }

    public float getSalestax()
    {
        return salestax;
    }

    public String getOrderref()
    {
        return orderref;
    }

    public String getReadytime()
    {
        return readytime;
    }

    public String getVendorextref()
    {
        return vendorextref;
    }

    public String getDeliverymode()
    {
        return deliverymode;
    }

    public ArrayList<OloOrdersSatusProduct> getProducts()
    {
        return products;
    }

    public boolean iseditable() {
        return iseditable;
    }

    public void setIseditable(boolean iseditable) {
        this.iseditable = iseditable;
    }
}

File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-Jamba-Android-v2.4.2/JMBA_Olo_iOS_SDK/Olo/OloOrderStatus.swift
//
//  OloOrderStatus.swift
//  Olo
//
//  Created by Taha Samad on 4/28/15.
//  Copyright (c) 2015 Hathway, Inc. All rights reserved.
//

import Foundation
import SwiftyJSON

public struct OloOrderStatus {
    
    public var id: String
    public var timePlaced: String
    public var vendorName: String
    public var status: String
    public var total: Double
    public var subTotal: Double
    public var salesTax: Double
    public var orderRef: String
    public var readyTime: String
    public var vendorExtRef: String
    public var deliveryMode: String
    public var deliveryAddress: OloDeliveryAddress?
    public var products: [OloOrderStatusProduct]
     public var isEditable: Bool
    public init(json: JSON) {
        id              = json["id"].stringValue
        timePlaced      = json["timeplaced"].stringValue
        vendorName      = json["vendorname"].stringValue
        status          = json["status"].stringValue
        total           = json["total"].doubleValue
        subTotal        = json["subtotal"].doubleValue
        salesTax        = json["salestax"].doubleValue
        orderRef        = json["orderref"].stringValue
        readyTime       = json["readytime"].stringValue
        vendorExtRef    = json["vendorextref"].stringValue
        deliveryMode    = json["deliverymode"].stringValue
        products        = json["products"].arrayValue.map { item in OloOrderStatusProduct(json: item) }
        deliveryAddress = OloDeliveryAddress(json: json["deliveryaddress"])
        
         isEditable      = json["iseditable"].boolValue
    }
    
}
